pkgs/development/libraries/qt-5/5.7/srcs.nix

```diff
@@ -122,6 +122,14 @@
       name = "qtimageformats-opensource-src-5.7.1.tar.xz";
     };
   };
+  qtinstaller = {
```

Comment:
Would  `qtinstaller`  not be overwritten, when this file is automatically regenerated?